pkgs/tools/typesetting/sile/default.nix

```diff
@@ -42,56 +44,67 @@ let
   ] ++ lib.optionals (lib.versionOlder lua.luaversion "5.3") [
     compat53
   ]);
-in
 
-stdenv.mkDerivation (finalAttrs: {
+  in stdenv.mkDerivation (finalAttrs: {
   pname = "sile";
-  version = "0.14.17";
+  version = "0.15.4";
 
   src = fetchurl {
-    url = "https://github.com/sile-typesetter/sile/releases/download/v${finalAttrs.version}/sile-${finalAttrs.version}.tar.xz";
-    sha256 = "sha256-f4m+3s7au1FoJQrZ3YDAntKJyOiMPQ11bS0dku4GXgQ=";
+    url = "https://github.com/sile-typesetter/sile/releases/download/v${finalAttrs.version}/sile-${finalAttrs.version}.tar.zst";
```

Comment:
Just out of curiosity, why change to `zst`?

Reply:
I've always wanted to use ZST, the reason for XV in the first place was that the GitHub Actions runners we were using to build the source had a version of GNU Autotools that did not support the zstd extension out of the box. Using XZ was less resistance. With more recent LTS of runners available by default that is no longer a limitation, and the generated configure scripts work on older systems as well so there were no remaining reasons *not* to use ZST which has in my mind superior properties.

Then the whole snafu with XZ being backdoored happened and we had folks complaining about the use of XZ. The complaints don't have much merit since the compromise did not affect generated archives, but since I'd always wanted ZST anyway just switching seemed like the easy move. I switched a bunch of my projects based on that logic.